Doc/c-api/complex.rst

```diff
@@ -25,13 +25,17 @@ pointers.  This is consistent throughout the API.
 
    The C structure which corresponds to the value portion of a Python complex
    number object.  Most of the functions for dealing with complex number objects
-   use structures of this type as input or output values, as appropriate.  It is
-   defined as::
+   use structures of this type as input or output values, as appropriate.
 
-      typedef struct {
-         double real;
-         double imag;
-      } Py_complex;
+   .. c:member:: double real
+   .. c:member:: double imag
+
+      It is defined as::
+
+         typedef struct {
+            double real;
+            double imag;
+         } Py_complex;
```

Comment:
Maybe just drop typedef?

```suggestion
   .. c:member:: double real

      real part

   .. c:member:: double imag
   
      imaginary part

```

Reply:
Yes, though with a complete sentence as the description of the two.

Reply:
I like that it shows all the details. Since it is rather compact, I would prefer to keep it.